and-bible/app/src/main/res/layout/speak_bible.xml

```diff
@@ -0,0 +1,328 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is good, but in the future you might like to experiment with ConstraintLayout which some people prefer for complex layouts.

Reply:
I'll read more about this at some point. (marking in my TODO)